### PR TITLE
Search project tech

### LINKED ISSUE #106: Search for projects by tech bubble tags

### PR DESCRIPTION
**Pull request by nickkoul**
Added functionality in the project search to search through the tech tags fixes #106

### CODE REVIEW

File: server/api/smallgroup/index.js [hunk 1]
```diff
@@ -12,6 +12,7 @@ router.get('/', controller.index);
 
 router.post('/', auth.hasRole('mentor'), controller.create);
 router.put('/:id', auth.hasRole('mentor'), controller.modify);
+router.put('/:id/name',controller.changeName);
```
Comment (agundy) [inline server/api/smallgroup/index.js]: Need auth on this endpoint!

**Comment by agundy** [comment]
It looks like you added your search changes on top of the edit small group name. You should break out just the individual commit.

**Comment by nickkoul** [comment]
Yeah I accidentally built it on the changes to edit small group name this should of fixed it. Got a little confused about how to roll it back with github but I believe this fixes it. Let me know if you see anything else

**Comment by agundy** [comment]
fixed it up for you!

**Pull request merged**